tests/acceptance-bdd/literature_suggestion.feature

```diff
@@ -0,0 +1,306 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+Feature: Litearature Suggestions
+    A web page to submit articles.
+
+  # arXiv text input
+  Scenario: Writing the correct arXiv id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id arxiv_id
+    Then I should not see the arxiv error message
+
+  Examples: Vertical
+    | input | 1001.4538 | hep-th/9711200 |
+
+  Scenario: Writing the wrong arXiv id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id arxiv_id
+    Then I should see the arxiv error message
+
+  Examples: Vertical
+    | input | hep-th.9711200 |
+
+  # DOI text input
+  Scenario: Writing the correct DOI id
```

Comment:
"DOI" stands for Digital Object Identifier, so you can drop the "id" here and in the following.